wizards/Argo/ArgoWizard.tsx

```diff
@@ -289,6 +289,25 @@ export function ArgoWizard(props: ArgoWizardProps) {
                             label="Repository type"
                             inputValueToPathValue={repositoryTypeToSource}
                             pathValueToInputValue={sourceToRepositoryType}
+                            onValueChange={(_, item: ApplicationSet) => {
+                                if (item.spec.template?.spec?.source.path !== undefined) {
```

Comment:
Typescript complained when I didn't use an **_if_** to conditionally check the item object before changing it's value, so I used the path/chart in an if statement to track the Git/Helm selection.